azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -45,6 +47,12 @@ def _run_op(self, op):
             self.sas_token = op.sas_token
             self.complete_op(op)
 
+        elif isinstance(op, pipeline_ops_base.SetCertificate):
```

Comment:
>SetCertificate [](start = 46, length = 14)

"Certificate" is too vague.  We should call this SetClientAuthenticationCertificate.  I'm also going to change "ca_cert" to trusted_certificate_chain or something like that.  This way we're very explicit about the purpose of the certificate.